lib/Twig/Loader/Filesystem.php

```diff
@@ -302,4 +302,34 @@ private function isAbsolutePath($file)
             || null !== parse_url($file, PHP_URL_SCHEME)
         ;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTemplateFreshness(Twig_CacheInterface $cache)
+    {
+        $cacheKey = $cache->generateKey(
+            md5(serialize($this->cache)),
+            get_class()
+        );
+
+        ob_start();
+        $cache->load($cacheKey);
+        $lastDate = ob_get_contents();
+        ob_end_clean();
+
+        if (!$lastDate) {
+            $lastModified = 0;
+            foreach ($this->cache as $name => $v) {
```

Comment:
I dont understand. As all the templates loaded can change the html output, it's what we need